starters/graphql-serverless-contentful/src/schema/post/post.resolvers.ts

```diff
@@ -0,0 +1,45 @@
+import { Resolvers } from '../../generated/graphql';
```

Comment:
Is there a way to silence these typescript errors? 

Everything is working fine of me locally but I didn't know if there is a way to silence those errors.

Reply:
Where are these errors Jessica, locally?

Reply:
These are the typescript errors I am seeing locally
<img width="1223" alt="Screen Shot 2022-11-22 at 2 17 31 PM" src="https://user-images.githubusercontent.com/67210629/203432609-31ca477b-ecea-49fd-b1cc-bf8a8b19ea02.png">

Reply:
Did you run codegen?

Reply:
I ran it now and the errors went away. 
I didn't realize I have to run that command each time.